### PR TITLE
Header

### PR DESCRIPTION
**Pull request by Marianna-Weychan**
Ok, finally managed to launch the browser so I'm catching up!
I removed the Navigation component and replaced it with a Header where I will place the navigation. In addition, once again I added svg icons (I corrected paths) and reinstalled node modules because I had wrong packages.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="pl">
+<html lang="en">
```
Comment (AdrianJar) [inline public/index.html:2]: It's very good to do this for future!

**Review by AdrianJar — COMMENTED** [review]
Good job with fixing this bug! And with job you've done so far!

**Review by AdrianJar — APPROVED** [review]
Good job with fixing this bug! And with job you've done so far!

**Comment by ghost** [comment]
@AdrianJar @KatarzynaZaloba 
I added the Header and after you approve the change, we will merge and we will have the first stage visible, so I am waiting for your feedbacks

File: src/core/App/theme.js [hunk 2]
```diff
@@ -7,13 +7,12 @@ export const theme = {
         pattensBlue: "#D6E4FF",
         mercury: "#E5E5E5",
         black: "#18181B",
```
Comment (KatarzynaZaloba) [inline src/core/App/theme.js:9]: Why white is deleted?

**Review by KatarzynaZaloba — APPROVED** [review]
Great job! Couldn't be better!

File: src/core/App/theme.js (hunk 2 above)
Comment (ghost) [reply src/core/App/theme.js:9]: because white was declared twice so I removed one

**Pull request merged**